package.json

```diff
@@ -7,6 +7,7 @@
     "restart-pm2": "pm2 restart all",
     "test": "NODE_ENV=test SERVER_PORT=6788 jest --forceExit",
     "start": "node bin/www",
```

Comment:
Do we need "devstart", "start", and "start-pm2"?

Normally in server apps, "start" is for production and "start:dev" is for development.

So,
start:dev: nodemon bin/www
start: pm2 start bin/www --name kleros-store